packages/clerk-js/src/core/clerk.ts

```diff
@@ -1615,9 +1615,11 @@ export class Clerk implements ClerkInterface {
     const userSignedIn = this.session;
     const signInUrl = this.#environment?.displayConfig.signInUrl;
```

Comment:
In ClerkJS both signInUrl and signUpUrl can be injected via ClerkOptions. These ClerkOptions come from CLERK_SIGN_IN_URL in frameworks such as Next.js. The display config settings are left as a fallback and are downplayed in v5.

So for this method, we should pick signInUrl and signUpUrl from ClerkOptions first.